tests/test_util.py

```diff
@@ -15,18 +15,27 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-import nose
 import os
 import numpy as np
 
-from pyfmi.fmi import FMUException, FMUModelME1, __ForTestingFMUModelME1, FMUModelCS1, load_fmu, \
-                      FMUModelCS2, FMUModelME2, __ForTestingFMUModelME2
+from pyfmi.fmi import (FMUException, FMUModelME1, _ForTestingFMUModelME1, FMUModelCS1, 
+                      FMUModelCS2, FMUModelME2, _ForTestingFMUModelME2)
 
 def get_examples_folder():
     return os.path.join(os.path.dirname(__file__), '..', 'examples')
 
-class Dummy_FMUModelME1(__ForTestingFMUModelME1):
-    # If true, makes use of the real __ForTesting implementation for nominal_continuous_states,
+class _ValuesDict(dict):
```

Comment:
Not super happy with this.

The core issue is that e.g., Dummy_FMUModelX.values could contain values that are e.g., np.array([1.]). 
This may be isolated to _ForTestingFMUModelX but goes back to various callbacks in the Test-problem definitions. However, fixing it there would be quite messy and non-instructive.

Reply:
Do we need a ticket to clean this up? I do agree it looks messy I wonder if we can improve it somehow, but then as a separate effort.